gap/small.gd

```diff
@@ -328,14 +362,31 @@ UnbindGlobal( "ID_AVAILABLE" );
 ##  <Func Name="ID_AVAILABLE" Arg='order'/>
 ##
 ##  <Description>
-##  returns false, if the identification routines for of groups of <A>order</A> is
-##  not installed. Otherwise a record with some information about the 
-##  identification of groups of <A>order</A> is returned.
+##  returns false, if the identification routines for groups of order
+##  <A>order</A> is not installed. Otherwise a record with some information
+##  about the identification of groups of order <A>order</A> is returned.
 ##  </Description>
 ##  </ManSection>
 ##
 DeclareGlobalFunction( "ID_AVAILABLE" );
 
+#############################################################################
+##
+#F  IdGroupsAvailable( <order> )
+##
+##  <#GAPDoc Label="IdGroupsAvailable">
```

Comment:
Hmm, maybe `IdGroupAvailable` or `HasIdGroup`, without the "s", to match `IsGroup`?

Reply:
`HasIdGroup` looks like a tester for an attribute - IMHO `IdGroupAvailable` is more consistent and also follows similar conventions elsewhere.

Reply:
Actually, upon re-reading this, I think `IdGroupsAvailable` (with an "s") is available, as IdGroup applies to a specific group, and this is about whether the id of all group*s* of given order are available.